src/components/MARCTable/MARCTable.js

```diff
@@ -60,6 +60,28 @@ export const MARCTable = ({
     setRows(updatedRows);
   };
 
+  const moveRow = (order, orderToSwitch) => {
+    const rowIndex = rows.findIndex(field => field.order === order);
+    const rowToSwitchIndex = rows.findIndex(field => field.order === orderToSwitch);
+    const updatedRows = [...rows];
+
+    updatedRows[rowIndex] = {
```

Comment:
If you store to a variable this and the statement below
```
const row = updatedRows[rowIndex] = {...
const updatedRow = updatedRows[rowToSwitchIndex] = {...

updatedRows[rowIndex] = updatedRow;
updatedRows[rowToSwitchIndex] = row;
```
The name of variables I suggested is probably not the best though.